File: cs50 - x/C/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h> 

int main(void)
{
    float change;
    do
    {
        change = get_float("change owed: ");
    }
    while (change <= 0.00);
    
    int cents = round(change * 100);
    int counter = 0;
    
    while (cents >= 25)
    {
        counter++;
        cents = cents - 25;
    }
    while (cents >= 10)
    {
        counter++;
        cents = cents - 10;
    }
    while (cents >= 5)
    {
        counter++;
        cents = cents - 5;
    }
    while (cents >= 1)
    {
        counter++;
        cents = cents - 1;
    }
    
    printf("%i\n", counter);
    
}

File: cs50 - x/nerya0001-cs50-problems-2020-x-sentimental-cash/nerya0001-cs50-problems-2020-x-sentimental-cash/cash.py
from cs50 import get_float

while True:
    amount = get_float("Change owed: ")
    if amount > 0.00:
        break

change = round(amount * 100)
count = 0

while change >= 25:
    count += 1
    change -= 25

while change >= 10:
    count += 1
    change -= 10

while change >= 5:
    count += 1
    change -= 5

while change >= 1:
    count += 1
    change -= 1

print (count)